xrpl4j-binary-codec/src/main/java/org/xrpl/xrpl4j/codec/binary/types/STObjectType.java

```diff
@@ -60,9 +60,24 @@ public STObjectType fromParser(BinaryParser parser) {
   public STObjectType fromJson(JsonNode node) {
     UnsignedByteArray byteList = UnsignedByteArray.empty();
     BinarySerializer serializer = new BinarySerializer(byteList);
+    boolean isUNLModify;
+    try {
+      isUNLModify = "UNLModify".equals(node.get("TransactionType").asText());
+    } catch (Exception e) {
+      isUNLModify = false;
+    }
+
 
     List<FieldWithValue<JsonNode>> fields = new ArrayList<>();
     for (String fieldName : Lists.newArrayList(node.fieldNames())) {
+
+      /**
+       * The Account field must not be a part of the UNLModify pseudotransaction encoding, due to a bug in rippled.
+       */
+      if (isUNLModify && "Account".equals(fieldName)) {
```

Comment:
Can we add a unit test for the case where `fieldName` is not `"Account"` but `isUNLModify` is true?

Reply:
This is actually happening for all other fields of the UNLModify transaction test that pass through here and get encodeed.